src/feature/client/transports.c

```diff
@@ -1858,7 +1858,9 @@ managed_proxy_stderr_callback(process_t *process,
   if (BUG(mp == NULL))
     return;
 
-  log_warn(LD_PT, "Managed proxy at '%s' reported: %s", mp->argv[0], line);
+  log_debug(LD_PT,
```

Comment:
Should this be log_info()?

I could imagine us wanting to see the logs when a PT user is having trouble. And if we ask for debug logs, they are usually hundreds of megabytes.

Reply:
Good catch. I've lowered it to info.